sky/utils/common_utils.py

```diff
@@ -13,13 +13,23 @@
 
 from sky import sky_logging
 
+_USER_HASH_FILE = os.path.expanduser('~/.sky/user_hash')
```

Comment:
I think this is a good change - in the future this also allows us to move the user to a new machine while preserving their hash